release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -596,6 +596,11 @@ The Metering Operator is deprecated as of {product-title} 4.6, and is scheduled
 [id="ocp-4-9-scale"]
 === Scale
 
+==== Special Resource Operator (Technology Preview)
```

Comment:
Remember to add a heading anchor: perhaps `[id="ocp-4-9-scale-special-resource-operator"]
`

Reply:
Done! Thanks for the reminder.